.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "Short feature description"
+labels: enhancement
+assignees: ''
+
+---
+
+Please fill out relevant sections below; remove those that are unused.
+
+**Feature Summary**
+Please give a clear, concise description of the proposed feature.  We are working on a [Minimal Viable Product](https://en.wikipedia.org/wiki/Minimum_viable_product), so please motivate why this feature is considered a high priority.
```

Comment:
Are we past MVP now? Might want to consider different language.

Reply:
Fixed